web/tests/firebase/firebase.js

```diff
@@ -1,25 +1,7 @@
+const config = require('../../main.config').firebaseConfig;
```

Comment:
I think library imports (`require('firebase')`, etc) are generally at the top?

Reply:
+1

also why are we using `require` over `import`? Isn't that `ES5` syntax? (i dont care im just wondering lol)

Reply:
looks like just following what was in this file before (see next line)

Reply:
Good points, I was just following what was in the file but I'll update it to ES6.